src/bin/slackApi.ts

```diff
@@ -71,8 +71,8 @@ async function getTextInternal(aiClient: AIClient, channel, input, reminders) {
 
 const SUPER_CLUSTER_USER_STRING = 'UVBBEEC4A';
 
-async function sendOotpChat(messages) {
-  axios
+async function sendOotpChat(messages, channel, say) {
+  return axios
```

Comment:
```suggestion
  const response = await axios
```

Reply:
I think you can remove the promise chains and just use `await` to make this a lot cleaner

Reply:
Done. I always forget about async/await.